docs/cli/commands.md

```diff
@@ -0,0 +1,561 @@
+plugin:diff              Diff an existing plugin with the creation template
+plugin:serve             Serves the dev/ folder of a plugin
+
+build                    Build a package for publishing
+build-workspace          Builds a temporary dist workspace from the provided packages
+lint                     Lint a package
+test                     Run tests, forwarding args to Jest, defaulting to watch mode
+clean                    Delete cache directories
+
+create-plugin            Creates a new plugin in the current repository
+remove-plugin            Removes plugin in the current repository
+
+config:print             Print the app configuration for the current package
+config:check             Validate that the given configuration loads and matches schema
+
+versions:bump            Bump Backstage packages to the latest versions
+versions:check           Check Backstage package versioning
+
+prepack                  Prepares a package for packaging before publishing
+postpack                 Restores the changes made by the prepack command
+
+help [command]           display help for command
+```
+
+## app:build
+
+Scope: `app`
+
+Builds a bundle of static content from the app, which can then be served via any
+static web server such as `nginx`, or via the
+[`app-backend`](https://www.npmjs.com/package/@backstage/plugin-app-backend)
+plugin directly from a Backstage backend instance.
+
+The command also reads and injects static configuration into the bundle. It is
+important to note that when deploying using your own static content hosting
+solution, this will be the final configuration used in the frontend unless you
+for example hook in configuration loading from the backend. When using the
+`nginx` image in this repo along with its included run script, `APP_CONFIG_`
+environment variables will be injected into the frontend, and when serving using
+the `app-backend` plugin, the configuration is completely injected from the
+backend and the configuration at the time of calling this command will not be
+used.
+
+Note that even when injecting configuration at runtime, it is not possible to
+change the base path of the app. For example, if you at build time have
+`app.baseUrl` set to `http://dev-app.com/my-app`, you can change that to
+`https://prod-app.com/my-app`, but not to `https://prod-app.com`, as that would
+change the path.
+
+During the build, the following variables are set:
+
+```java
+process.env.NODE_ENV = 'production';
+process.env.BUILD_INFO = {
+  cliVersion: '0.4.0', // The version of the CLI package
+  gitVersion: 'v0.4.0-86-ge54815618', // output of `git describe --always`
+  packageVersion: '1.0.5', // The version of the app package itself
+  timestamp: 1678900000000, // Date.now() when the build started
+  commit: 'e548156182a973ed4b459e18533afc22c85ffff8', // output of `git rev-parse HEAD`
+};
+```
+
+Some CI environments do not properly report correct resource limits, potentially
```

Comment:
👍 Here's the official link to the underlying explanation and config option just in case it makes sense to provide as well - https://github.com/webpack-contrib/terser-webpack-plugin#parallel

Reply:
Thanks!